highfidelity/hifi
# Added login button to Tablet

https://highfidelity.fogbugz.com/f/cases/6874/Add-Login-Button-to-Tablet

Test plan:
- Open Tablet
- log out and open Tablet again
- Observe "Log in" in top right corner
- Press it and do a log in
- Observe user name in top right corner of the Tablet
